rumqttc/src/v5/mod.rs

```diff
@@ -53,31 +54,27 @@ pub struct MqttOptions {
     /// keep alive time to send pingreq to broker when the connection is idle
     keep_alive: Duration,
     /// clean (or) persistent session
-    clean_session: bool,
+    clean_start: bool,
     /// client identifier
     client_id: String,
     /// username and password
     credentials: Option<(String, String)>,
-    /// maximum incoming packet size (verifies remaining length of the packet)
-    max_incoming_packet_size: usize,
-    /// Maximum outgoing packet size (only verifies publish payload size)
-    // TODO Verify this with all packets. This can be packet.write but message left in
-    // the state might be a footgun as user has to explicitly clean it. Probably state
-    // has to be moved to network
-    max_outgoing_packet_size: usize,
     /// request (publish, subscribe) channel capacity
     request_channel_capacity: usize,
     /// Max internal request batching
     max_request_batch: usize,
     /// Minimum delay time between consecutive outgoing packets
     /// while retransmitting pending packets
     pending_throttle: Duration,
-    /// maximum number of outgoing inflight messages
-    inflight: u16,
     /// Last will that will be issued on unexpected disconnect
     last_will: Option<LastWill>,
     /// Connection timeout
     conn_timeout: u64,
+    /// Default value of for maximum incoming packet size.
+    /// Used when `max_incomming_size` in `connect_properties` is NOT available.
```

Comment:
`The ConnectionProperties` struct doesn't have `max_incomming_size`.
Instead, it has `max_packet_size`.

To avoid confusion, this comment should be updated to reflect this information.